libs/ui/packages/native/src/components/Form/Input/AnimatedInput/index.tsx

```diff
@@ -46,7 +46,13 @@ const AnimatedInput = (
 
   return (
     <InputContainer status={inputStatus} style={style}>
-      {placeholder && <AnimatedNotchedLabel placeholder={placeholder} inputStatus={inputStatus} />}
+      {placeholder && (
+        <AnimatedNotchedLabel
+          placeholder={placeholder}
+          inputStatus={inputStatus}
+          value={value as string}
```

Comment:
Why do we need to use `as` here? Can we manage to have the correct type and avoid using `as`?

Reply:
it's already a string indeed no need to use the "as"